ksqldb-serde/src/main/java/io/confluent/ksql/serde/KeyFormatUtils.java

```diff
@@ -25,7 +25,8 @@
       FormatFactory.NONE,
       FormatFactory.KAFKA,
       FormatFactory.DELIMITED,
-      FormatFactory.JSON
+      FormatFactory.JSON,
+      FormatFactory.JSON_SR
```

Comment:
How come we're making JSON_SR available by default, given that AVRO still isn't enabled by default? Aren't both blocked on the same additional testing for joins?

Also, what's the point of having `"ksql.key.format.enabled": true` in the new QTTs (with primitive keys) if we're enabling this by default?

Reply:
🤦 this is me not being careful. Thank you for checking this